src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs

```diff
@@ -173,6 +174,22 @@ private void RawKeyDown(RawKeyEventArgs e)
 
                     break;
                 }
+
+                case RawInputModifiers.Control | RawInputModifiers.Alt when e.Key == Key.F:
+                {
+                    vm.FreezePopups = !vm.FreezePopups;
+
+                    foreach (var popupRoot in GetPopupRoots(Root))
+                    {
+                        if (popupRoot.Parent is Popup popup)
+                        {
+                            popup.IsLightDismissEnabled = !vm.FreezePopups;
```

Comment:
IsLightDismissEnabled is a styled property. 
If app developers did set value with an style priority, any value set by dev tools will have higher (local) priority. So it will break some apps. 
As a workaround, I think, setting "True"  value with Animation priority (which is higher than local) and disposing previous setters on "False" value (i.e. "reset") will work. 

Also in long term we probably can have "diagnostic" priority value, can be discussed later. I see some other benefits of it.

Reply:
Or probably you can do the same, but with LocalValue priority, set by SetValue method and disposing result, when we don't need it.

Reply:
Calling `popup.SetValue(Popup.IsLightDismissEnabledProperty, !vm.FreezePopups)` returns `null` for me.